### DISCUSSION TITLE
Error [ERR_UNSUPPORTED_ESM_URL_SCHEME], App crashes after upgrading to 14.2.0 PLEASE HELP

### DISCUSSION THREAD

**Discussion by 1pharaxh** [vercel/next.js#64510] [App Router] [answered]
I get this error after upgrading from next 14.1.0 to 14.2.1 my web app works fine without the --turbo flag but I was just curious what is causing it to crash now since it worked fine with 14.1.0 with --turbo flag enabled on dev. are there any fixes / workarounds to this ? Also I am running next on windows 11 node v20.6.1

Please help me 🙏🏽

Thank you

<img width="955" alt="image" src="https://github.com/vercel/next.js/assets/93630550/4e89102b-1fd1-4c62-92d5-56d85128add3">

**Comment by SidKH** [comment, marked as answer]
Hey 👋

The fix to this [has been merged to canary](https://github.com/vercel/next.js/pull/64386), so it should be available soon.
In the meantime, you can try [this workaround](https://github.com/vercel/next.js/issues/64371#issuecomment-2050735468)
